Parser/pegen/pegen.c

```diff
@@ -388,14 +347,14 @@ tokenizer_error(Parser *p)
 }
 
 void *
-_PyPegen_raise_error(Parser *p, PyObject *errtype, const char *errmsg, ...)
+_PyPegen_raise_error(Parser *p, PyObject *errtype, int no_col_number, const char *errmsg, ...)
```

Comment:
I think I'd rename `no_col_number` to something like `with_col_number` and change the definitions of `RAISE_SYNTAX_ERROR[_NO_COL_OFFSET]` to pass the correct values. It'll be more clear in the `if`s that follow this way. At least for me, it's easier to quickly understand `if (with_col_number)` rather than `if (!no_col_number)`.

Reply:
Sure, good idea.